### PR TITLE
add restake on migaloo, terra2, kuji, osmosis

### PR DESCRIPTION
**Pull request by nick134-bit**

### CODE REVIEW

File: kujira/assetlist.json [hunk 1]
```diff
@@ -1107,6 +1107,54 @@
...
+        },
+        {
+          "denom": "rstk",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/82B298F9E74A9A211C55C67C7AB0C790836BD43672D5DD830F7AE2C5D010AD7D",
+      "name": "Restake DAO Token",
+      "display": "RESTAKE",
```
Comment (JeremyParish69) [inline kujira/assetlist.json:1128]: Please fix the "display" value for each asset. They must refer to a denom_unit. It this case, it should be "rstk". This applies to all the assets.

File: migaloo/assetlist.json [hunk 2]
```diff
@@ -499,6 +499,33 @@
...
+    },
+    {
+      "description": "Restake DAO Token",
+      "denom_units": [
+        {
+          "denom": "factory/migaloo1d0uma9qzcts4fzt7ml39xp44aut5k6qyjfzz4asalnecppppr3rsl52vvv/rstk",
+          "exponent": 0
+        },
+        {
+          "denom": "RSTK",
```
Comment (JeremyParish69) [inline migaloo/assetlist.json:511]: Please change to 'rstk' so the display value matches to a denom unit

**Comment by JeremyParish69** [comment]
As per the error found by the validation check, the display denom is referring to multiple denom units.
Can you please replace the base denom unit's alias 'rstk' with 'factory/migaloo1d0uma9qzcts4fzt7ml39xp44aut5k6qyjfzz4asalnecppppr3rsl52vvv/rstk'

**Comment by nick134-bit** [comment]
@JeremyParish69 thank you very much for walking me through this

**Pull request merged**